viewer/src/migration/NodeIdAndTreeIndexMaps.ts

```diff
@@ -46,4 +54,33 @@ export class NodeIdAndTreeIndexMaps {
     });
     return result;
   }
+
+  getNodeId(treeIndex: number): number | undefined {
```

Comment:
Why does `getNodeId()` return `number | undefined` while `getTreeIndex` returns a `Promise`? I suggest `getNodeId` is renamed to `tryGetNodeId` if it cant be a promise for some reason.

Reply:
Because we cannot ask the API for a node ID from a tree index currently. I agree that we can rename it to tryGetNodeId.